src/test/java/org/openRealmOfStars/player/combat/CombatTestByDiocto.java

```diff
@@ -0,0 +1,544 @@
+package org.openRealmOfStars.player.combat;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.openRealmOfStars.player.combat.*;
+import org.openRealmOfStars.player.PlayerInfo;
+import org.openRealmOfStars.player.fleet.Fleet;
+import org.openRealmOfStars.player.fleet.FleetList;
+import org.openRealmOfStars.player.ship.Ship;
+import org.openRealmOfStars.player.ship.ShipComponent;
+import org.openRealmOfStars.player.ship.ShipStat;
+import org.openRealmOfStars.starMap.Coordinate;
+import org.openRealmOfStars.starMap.Route;
+
+import static org.mockito.Mockito.*;
+import org.mockito.Mockito;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+/**
+*
+* Open Realm of Stars game project
+* Copyright (C) 2017  Tuomo Untinen
```

Comment:
Since this is your original file. You could have your own copyright here. If someone adds new test case in same file
then she or he could add his own copyright text here.